locale/en/about/community.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Community Committee
+layout: about.hbs
+---
+
+# Community Committee
+
+The Community Committee (CommComm) is a top-level committee in the Node.js Foundation focused on community-facing efforts. Code commits !== the only way to contribute!
+
+The CommComm has authority over community outreach including:
+ - Community Evangelism
+ - Education Initiatives
+ - Cultural Direction of Node.js Foundation
+ - Community Organization Outreach
+ - Project Moderation/Mediation
+ - Public Outreach and [Publications](https://medium.com/the-node-js-collection)
+
+There are four levels of engagement with the Community Committee:
```

Comment:
Don't know if this is the right verbiage. Engagement? Commitment? Involvement?

Reply:
I'd go with involvment, engagement seems too definitive for me

Reply:
I'd add in Internationalization Working Group info

Reply:
+1 to involvement over engagement 👍